### PR TITLE
LIVE-896 - LLD market UI improvments

### PR DESCRIPTION
**Pull request by nparigi-ledger**
:warning: Don't merge before https://github.com/LedgerHQ/ui/pull/173 ! :warning:

## 🦒 Context (issues, jira)

https://ledgerhq.atlassian.net/browse/LIVE-896
https://ledgerhq.atlassian.net/browse/LIVE-969

## 💻  Description / Demo (image or video)
Changes following this figma feedback https://www.figma.com/file/I1ySQLJIx8AhlSVqF6WFos/LLD-%2F-Market?node-id=1148%3A54421

- Change sidebar dot style 

Market page : 
- Show change range
- change filters style
- change not found image
- Market item row "change" empty state

## 🖤  Expectations to reach

PR must pass CI, rebase develop if conflicts. Thanks!

- **on QA**: at least one of these two checkboxes must be checked:
  - [ ] a specific test planned is defined on Jira
  - [x] this PR is covered by automatic UI test
- **on delivery**: at least one of these two checkboxes must be checked: 
  - [ ] Option 1: **no impact**: The changes of this PR have ZERO impact on the userland (invisible for users)
  - [x] Option 2: **atomic delivery**: the changes is atomic and complete (no partial delivery)

### CODE REVIEW

File: src/renderer/screens/market/DropDown.tsx [hunk 1]
```diff
@@ -30,12 +30,16 @@ function DropdownControl<O, M extends boolean, G extends GroupBase<O>>(
...
 function DropdownValueContainer<O>(props: ValueContainerProps<O, false>) {
   const ChevronIcon = props.selectProps.menuIsOpen ? ChevronTopMedium : ChevronBottomMedium;
+  const { label } = (props.selectProps as unknown) as Props<O>;
 
   return (
     <ValueContainer
       {...props}
       render={() => (
         <FlexBox alignItems="center" flexDirection="row">
+          <Text fontWeight="semiBold" variant={"paragraph"} color="neutral.c80" mr={2}>
```
Comment (ofreyssinet-ledger) [inline src/renderer/screens/market/DropDown.tsx:40]: not important at all but for the style nazi that I am, i'm curious why sometimes you usa string props like that `fontWeight="semiBold"` and sometimes like that `variant={"paragraph"}` haha

Comment (ofreyssinet-ledger) [reply src/renderer/screens/market/DropDown.tsx:40]: (imo the first is cleaner)

File: src/renderer/screens/market/MarketCoinScreen/index.tsx [hunk 2]
```diff
@@ -206,10 +205,10 @@ export default function MarketCoinScreen() {
             )}
           </CryptoCurrencyIconWrapper>
           <Flex pl={3} flexDirection="column" alignItems="left" pr={16}>
-            <Flex flexDirection="row" alignItems="center">
+            <Flex flexDirection="row" alignItems="center" justifyContent={'center'}>
```
Comment (ofreyssinet-ledger) [inline src/renderer/screens/market/MarketCoinScreen/index.tsx:208]: 👀

**Review by OlivierFreyssinet — APPROVED** [review]

File: src/renderer/screens/market/DropDown.tsx (hunk 1 above)
Comment (nparigi-ledger) [reply src/renderer/screens/market/DropDown.tsx:40]: It's just my IDE :safety_vest:
I agree it's prettier yes but i think we don't care also ? The perfect solution would be a lint rule

**Review by LFBarreto — APPROVED** [review]

**Review by ychen-ledger — COMMENTED** [review]
Validated https://ledgerhq.atlassian.net/browse/LIVE-897

**Review by ychen-ledger — CHANGES_REQUESTED** [review]
update search input introduced a new issue, please see detail in https://ledgerhq.atlassian.net/browse/LIVE-898

**Review by ychen-ledger — COMMENTED** [review]
Validated with limitation of ui lib with the v3 theme https://ledgerhq.atlassian.net/browse/LIVE-896

**Review by ychen-ledger — CHANGES_REQUESTED** [review]
@nparigi-ledger Plz check there conflicts and no artifact generated🙏

**Review by ychen-ledger — CHANGES_REQUESTED** [review]
@nparigi-ledger It seems an regression issue that clear button is missing in the new local build 🥶
Screenshot 2022-03-09 at 17 56 34

**Review by ychen-ledger — APPROVED** [review]
Validated on: 

- MacOs 
- LLD 2.39.2
- https://ledgerhq.atlassian.net/browse/LIVE-898

**Pull request merged**